react-components/jsx.md

```diff
@@ -0,0 +1,321 @@
+# JSX
+
+## Goals
+
+React JS can use JavaScript syntax known as JSX. It's used when rendering components in our webapp, so we can expect to see a lot of JSX!
+
+JSX is used to declare HTML-like UI in JavaScript. JSX determines what our webapp's rendered HTML looks like.
+
+Before we use JSX extensively, let's pause to understand what JSX is and how to write it.
+
+## Introduction
+
+In a newly created React app using `create-react-app`, we can see the following code snippets:
+
+`src/index.js`
+
+```js
+<React.StrictMode>
+  <App />
+</React.StrictMode>
+```
+
+`src/App.js`
+
+```js
+<div className="App">
+  <header className="App-header">
+    <img src={logo} className="App-logo" alt="logo" />
+    <p>
+      Edit <code>src/App.js</code> and save to reload.
+    </p>
+    <a
+      className="App-link"
+      href="https://reactjs.org"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Learn React
+    </a>
+  </header>
+</div>
+```
+
+What is this code? How do we read it? How do we write it?
+
+## JSX
+
+[**JSX**](https://reactjs.org/docs/introducing-jsx.html) is a _syntax extension_ to JavaScript. It is JavaScript code that isn't normally available in vanilla JS projects.
+
+In our React projects, we'll write the logic for _how_ to render our components in JSX.
```

Comment:
i think it reads a bit more easily if we combine the two sentences about the syntax. And I don't know why, but I like the idea of adding this sneaky note about jsx really being js, since it is, from a certain point of view.
```suggestion
[**JSX**](https://reactjs.org/docs/introducing-jsx.html) is a _syntax extension_ to JavaScript that isn't normally available in vanilla JavaScript projects. It may look like HTML, but it's really JavaScript _in disguise_!

In our React projects, we'll write the logic for _how_ to render our components using JSX.
```